module13and14/LoopCanvas/LoopCanvas/Views/HomeView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  HomeView.swift
+//  LoopCanvas
+//
+//  Created by Peter Rice on 6/25/24.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+  @StateObject var canvasViewModel = CanvasViewModel(
+    canvasModel: CanvasModel(), musicEngine: AudioKitMusicEngine())
+
+  var body: some View {
+    NavigationStack {
+      List {
+        NavigationLink(value: "New Song") {
+          HStack {
+            Text("New Song")
+          }
+        }
+        NavigationLink(value: "All Songs") {
+          HStack {
+            Text("All Songs")
+          }
+        }
+        NavigationLink(value: "Recents") {
+          HStack {
+            Text("Recents")
+          }
+        }
+        NavigationLink(value: "Shared") {
+          HStack {
+            Text("Shared")
+          }
+        }
+        NavigationLink(value: "Favorites") {
+          HStack {
+            Text("Favorites")
+          }
+        }
+      }
+      .navigationDestination(for: String.self) { selection in
```

Comment:
This is not really going to be maintanable, and it relies on your not making typos. A format of 
````
NavigationLink(destination: PlaceHolderView()) {
  HStack {
    Text("Shared")
  }
}
````
is going to let the compiler be able to find mistakes and is going to make it easier to modify this list in the future.